From 35280f5830cb2d6ef63bbcc371260a0999bd898e Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 18 Aug 2015 13:07:19 +0300
Subject: [PATCH 0050/1345] ARM: mvebu: Add at24c64 EEPROM to A388-DB DTS

commit  91d4374e5d731c45b425e460e83be645ea15054b from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Add at24c64 i2c EEPROM at address 0x54 to i2c-0 bus
  in armada-388-db.dts file
- Once the driver enabled, the EEPROM content can be
  accessed through the sysfs entry:
  /sys/bus/i2c/devices/0-0054/eeprom
  * For instance the following command will print the HEX
    dump of the EEPROM:
    hexdump -C /sys/bus/i2c/devices/0-0054/eeprom
  * The next command will write 16 bytes from file "test"
    to offset 0 of the EEPROM
    dd if=./test of=/sys/bus/i2c/devices/0-0054/eeprom bs=1 count=16

Change-Id: I6f0e6fe66fe3ed7dfbd32abd8956590bfa288ede
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22909
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/23053
Tested-by: Star_Automation <star@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27573
Reviewed-by: Lior Amsalem <alior@marvell.com>
Tested-by: Lior Amsalem <alior@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/armada-388-db.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/armada-388-db.dts b/arch/arm/boot/dts/armada-388-db.dts
index 332a41e..5592288 100644
--- a/arch/arm/boot/dts/armada-388-db.dts
+++ b/arch/arm/boot/dts/armada-388-db.dts
@@ -78,6 +78,12 @@
 					compatible = "cirrus,cs42l51";
 					reg = <0x4a>;
 				};
+
+				eeprom@54 {
+					compatible = "atmel,24c64";
+					pagesize = <32>;
+					reg = <0x54>;
+				};
 			};
 
 			i2c@11100 {
-- 
1.7.9.5

